projects/novo-elements/src/elements/date-picker/DatePickerInput.ts

```diff
@@ -234,6 +223,10 @@ export class NovoDatePickerInputElement implements OnInit, ControlValueAccessor
       if (!value) {
         return '';
       }
+      // Handle dates stored as string values
+      if (/\d{4}-\d{2}-\d{2}/.test(value)) {
```

Comment:
Does `dateFns` not respect `YYYY-MM-DD` as a valid format? If it does, then this is essentially allowing this one format without having to pass a `userDefinedFormat`? And if we need this, what happens if it's `YYYY/MM/DD` or `YYYY\MM\DD`? And I'm pretty sure a value like `2012-99-00` is will come back as valid with this.

Reply:
yes, and i just remembered that we have a version of the date-picker with text-mask enabled which allows only for `/` as a de-limiter. and the date-format is defined in the translation files. if we want to handle string values, why aren't we using the text mask?